go/vt/wrangler/keyspace.go

```diff
@@ -850,7 +850,7 @@ func (wr *Wrangler) DeleteKeyspace(ctx context.Context, keyspace string, recursi
 		wr.Logger().Infof("Deleting all shards (and their tablets) in keyspace %v", keyspace)
 		for _, shard := range shards {
 			wr.Logger().Infof("Recursively deleting shard %v/%v", keyspace, shard)
-			if err := wr.DeleteShard(ctx, keyspace, shard, recursive); err != nil && err != topo.ErrNoNode {
+			if err := wr.DeleteShard(ctx, keyspace, shard, true /*recursive*/, true /*ignoreIfServing*/); err != nil && err != topo.ErrNoNode {
```

Comment:
s/ignoreIfServing/evenIfServing/

And it's better to have spaces around parameter names (i.e. "/* recursive */").